README.md

```diff
@@ -76,3 +76,68 @@ Make sure you read the `README.md` file in the `reporting` directory and install
 ## Implementing the benchmark for a new tool
 
 To implement a tool, you need to create a new directory in the solutions directory and give it a suitable name.
+
+## Running the benchmark with Docker
+
+Instructions for running the benchmark starting with a fresh Ubuntu 20.04 VM.
+
+### Configuration
+
+Create a file system and mount it. Example:
+
+```bash
+sudo mkfs.ext4 /dev/nvme1n1
```

Comment:
This assumes that such a device exists. Should we mention above that this would be running on a specific type of VM in a specific cloud provider?

Reply:
Added some clarification for it.

Reply:
Thanks, but I meant more like when is this going to matter :-D. In a normal dev workstation you wouldn't have to do this, I think? I certainly don't have a `/dev/nvme1n1` device in my machine.

Reply:
Marton pushed a comment which explains that this is:
> In case of Amazon EC2